main_test.go

```diff
@@ -26,6 +26,7 @@ func Test_getRepo(t *testing.T) {
 		{"ssh with scheme without port", "ssh://user@github.com/owner/repository.git", "owner/repository", false},
 		{"ssh with scheme without port, no user", "ssh://github.com/owner/repository.git", "owner/repository", false},
 		{"ssh with scheme with port", "ssh://user@github.com:22/owner/repository.git", "owner/repository", false},
+		{"ssh with scheme with mutilple path", "ssh://gitlab.company.com:category/project-name/subproject/finalname.git", "owner/repository", false},
```

Comment:
I think the expected string was not on pair with the input. Did you wanted to something similar to this?
```suggestion
		{"ssh with scheme with mutilple path", "ssh://gitlab.company.com:category/project-name/subproject/repository.git", "project-name/subproject/repository", false},
```